File: app/src/main/java/br/com/rvvaranda/hu/Model/Price.kt
package br.com.rvvaranda.hu.Model


import com.google.gson.annotations.SerializedName

data class Price(
    @SerializedName("amount")
    val amount: Double,
    @SerializedName("amountPerDay")
    val amountPerDay: Double,
    @SerializedName("currency")
    val currency: String,
    @SerializedName("currency_original")
    val currencyOriginal: String,
    @SerializedName("current_price")
    val currentPrice: Double,
    @SerializedName("fee_extra")
    val feeExtra: Int,
    @SerializedName("fee_extra_original")
    val feeExtraOriginal: Int,
    @SerializedName("gain")
    val gain: Int,
    @SerializedName("gain_original")
    val gainOriginal: Int,
    @SerializedName("old_price")
    val oldPrice: Double,
    @SerializedName("originalAmountPerDay")
    val originalAmountPerDay: Double,
    @SerializedName("sku")
    val sku: String,
    @SerializedName("tariff_policies")
    val tariffPolicies: ArrayList<Any>,
    @SerializedName("taxes")
    val taxes: ArrayList<Taxe>,
    @SerializedName("total_price")
    val totalPrice: Double
)

File: HurbHotel_iOS/HurbHotel_iOS/Commons/Model/Price.swift
//
//  Price.swift
//  HurbHotel_iOS
//
//  Created by Helio Junior on 22/07/20.
//  Copyright © 2020 HelioTecnologia. All rights reserved.
//

import Foundation

struct Price: Decodable {
    let amount: Double?
    let amountPerDay: Double?
    let currency: String?
    let currency_original: String?
    let current_price: Double?
    let fee_extra: Double?
    let fee_extra_original: Double?
    let gain: Double?
    let gain_original: Double?
    let old_price: Double?
    let originalAmountPerDay: Double?
    let sku: String?
    let total_price: Double?
}
